homeassistant/components/goalzero/const.py

```diff
@@ -129,7 +129,7 @@
         ATTR_DEFAULT_ENABLED: True,
     },
     "timestamp": {
-        ATTR_NAME: "Up Time",
+        ATTR_NAME: "Total Run Time",
```

Comment:
Shouldn't we mark it as a breaking change?

Reply:
No. The sensor platform just got merged. It only lives in dev.